edge/pkg/devicetwin/dtmanager/twin.go

```diff
@@ -23,8 +23,6 @@ const (
 	RestDealType = 0
 	//SyncDealType update form cloud sync
 	SyncDealType = 1
-	//DetailDealType detail update from cloud
-	DetailDealType = 2
```

Comment:
We should not delete this one, looks like it's continuous with `SyncDealType` and `SyncTwinDeleteDealType `

Reply:
Yes, the DetailDealType may be used later in bussiness and ut code. Has been changed back.